### PR TITLE
[WIP] Replaces Syndicate thrown weapons in gambling drop table with toy versions

### LINKED ISSUE #47792: Metastation roundstart bar/armory throwing kit box has illegal tech
Syndicate boxes have illegal tech for some reason, it's likely a bad thing that illegal tech spawns roundstart on metastation

### PR DESCRIPTION
**Pull request by TheVekter**
Fixes #47792, change is because this makes it far too easy to research illegal tech on the rounds it spawns in.

Instead, I replaced them with toy versions of the throwing stars and bola. The stars will still embed but do zero damage, do not make you bleed, and can be removed easily. The bola will still slow you but has an incredibly short removal time.

Also added a way to craft the syndicate bundle boxes by request.

I also removed the contraband gun spawner from the armory, as it doesn't seem very useful aside from "HoP/Captain that wants to validhunt". I might revert this if it proves to be controversial enough.

## Changelog
:cl:
add: You can now craft a dangerous items box (the box Syndicate bundles come in) using one sheet of cardboard.
add: DonkSoft has introduced a new line of thrown weapons!
balance: Syndicate items can no longer drop from the "Gambling items" or "Armory contraband" loot tables.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
dude, just remove these items from the list of items that you can gain syndicate tech from

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 1]
```diff
@@ -40,29 +40,28 @@
 				/obj/item/gun/ballistic/automatic/pistol = 8,
 				/obj/item/gun/ballistic/shotgun/automatic/combat = 5,
 				/obj/item/gun/ballistic/revolver/mateba,
-				/obj/item/gun/ballistic/automatic/pistol/deagle
+				/obj/item/gun/ballistic/automatic/pistol/deagle,
```
Comment (tralezab) [inline code/game/objects/effects/spawners/lootdrop.dm]: comma at the end of a list, i think that's a big wee woo problem

**Comment by TheVekter** [comment]
> dude, just remove these items from the list of items that you can gain syndicate tech from

From my understanding of it, that's a hard-coded function of all Syndicate uplink items. Not quite as simple as that. Having roundstart syndicate items is not a good idea, especially in such easy reach of the clown honk

**Comment by TheVekter** [comment]
I went ahead and added the Ushanka as another drop option as the odds left it as "just the Russian revolver and sometimes maybe a coin".

**Comment by py01** [comment]
The property that makes items have illegal tech is in all_nodes.dm, line 1048. You can add illegal_tech = FALSE to uplink item datums to remove their illegal tech property.

**Comment by TheVekter** [comment]
> The property that makes items have illegal tech is in all_nodes.dm, line 1048. You can add illegal_tech = FALSE to uplink item datums to remove their illegal tech property.

I kind of don't actually want to do this because it removes an option from what's already moderately difficult to get. It's better to just say "This can't spawn out in the open where anyone can get it" than to remove that functionality just because it sometimes shows up in the bar.

**Comment by wesoda25** [comment]
I like this. Non antags shouldn't be able to get items as strong as the throwing kit, I know I've gotten a ton of bullshit kills on other people (and myself) through them. As for the cards, it makes them more interesting when actually used and less of a boring commodity.

**Comment by ATH1909** [comment]
Could you add the ability to craft cardboard into syndicate boxes, then?

**Comment by TheVekter** [comment]
> Could you add the ability to craft cardboard into syndicate boxes, then?

The one with the red borders? I don't see why not. I'll do it in another PR.

**Comment by Pugie** [comment]
You forgot to remove stechkin from Meta contraband loot.

**Comment by Shadowflame909** [comment]
I don't like this. A part of the reason these things can just appear on station like this is to break the meta of "Only antags can have certain things."

Let me tell you, virtually any item is obtainable while being a non-antag. Through good luck, RNG, and badmins having mercy on you.

You can decide "hey syndicate items having a chance to exist round start? No-way?"

But every time you do so. A maintainer cries because you've just inflated the value of these "antag points" even more.

Tldr; Let me ride in my space looted nukie hardsuit with my sketchin pistol, wizard magic, and holoparasite without anyone instantly assuming antag you anti-paranoia gamer!!!

Tldr 2: At least replace them with the toy variants so the same amount of paranoia still exists in the game. Or unusable items like cult bolas.

**Comment by TheVekter** [comment]
I actually 100% agree with you. More opportunities for meta-breaking bullshit is always good. This is why I added the suspicious box as something you can just craft. That being said, having items that can pretty reliably murder someone in three seconds just sitting out probably isn't in best form.

I'm contemplating doing something like foam shurikens or foam bolas as opposed to outright removing them from the map, but I'll work on that later tonight.



> You forgot to remove stechkin from Meta contraband loot.

Good catch.

**Comment by Pugie** [comment]
Plastic throwing star embeds that do no damage but prank people could be funny.

**Comment by Tlaltecuhtli** [comment]
if you want it to not give syndi tech if roundstart item you can just add an if(mapload) synditech=0 on initalize

**Comment by carlarctg** [comment]
the same retarded captain going to the armory every round to take the stetchkin/357 and mass craft 10mm ammo boxes/357 rounds isn't 'hard' it's just stupid

**Comment by TheVekter** [comment]
Carlarc has a point also don't merge this

**Comment by TheVekter** [comment]
This is ready to merge assuming I didn't fuck anything up.

**Comment by TheVekter** [comment]
Going to close this for now. Ryll is working on making embedding a component which would let me un-fuck this code. I'll do something else for a temp fix and do this part later.
